### PR TITLE
WIP: Fix/naming issue, next regimen is 1.1.2 

### PR DESCRIPTION
**Pull request by jeromelecoq**
1/ This is make stash and GitHub more in par
2/ Correcting naming issue

### CODE REVIEW

File: regimen.yml [hunk 1]
```diff
@@ -137,7 +129,7 @@ _image_set_d_parameters: &image_set_d_parameters
 
 ### mtrain definitions
 
-name: VisualBehavior_Task1A_v1.0.3
+name: VisualBehavior_Task1A_v1.1.1
```
Comment (nicain) [inline regimen.yml:132]: This regimen name is already deployed.

**Review by nicain — CHANGES_REQUESTED** [review]

**Comment by nicain** [comment]
This branch is currently 2 commits past the tagged 1.1.1 branch that was deployed (64d848e4982e3edc5384b2699f1ae2ced1ac42b4). My suggestion is to use this branch, with an additional commit, to define regimen 1.1.2. @jeromelecoq does this sound good?

**Comment by jeromelecoq** [comment]
ok

**Comment by nicain** [comment]
@jeromelecoq I put some testing infrastructure in-place, so that the current problem with this PR causes a blocker to merging.

You can read more about the infrastructure here:
https://github.com/AllenInstitute/mtrain_regimens/issues/67

This PR will not be mergable until, in this case, the regimen-name-already-in-use problem is fixed.  To see that this is the problem, go to "details" and click down through the build report until you hit the error report: http://bamboo.corp.alleninstitute.org/browse/VB-MB3-JOB1-1/test/case/169970754.  A new commit on the branch that is the source of the PR can fix the issues; the continuous-integration will be re-run for every commit.

**Pull request closed without merging**